src/assets/vocabulary/css/vocabulary.css

```diff
@@ -298,6 +298,8 @@ button.expand-menu {
 
 .ancilliary-menu ul li button:hover {
     cursor: pointer;
+    background-color: #FBE9E7;
+    color: #000;
```

Comment:
changes cannot be made directly to `vocabulary.css` in downstream projects, and should instead be made as override styles within the local `style.css` file.